MotorControl/src/main/kotlin/org/sagebionetworks/motorcontrol/presentation/TremorStepFragment.kt

```diff
@@ -59,6 +65,8 @@ open class TremorStepFragment: StepFragment() {
 
     private lateinit var step: TremorStepObject
 
+    private var textToSpeech: TextToSpeech? = null
```

Comment:
I think textToSpeech could be done as a lateinit var to insure that it should never be null.